### ISSUE TITLE
Is it possible to point monolith at local files?

### ISSUE THREAD

**Issue by mfaughn** [Y2Z/monolith#180]
Just wondering if it is possible. It would be nice if it could. I'm trying it and getting an error.

```
$ RUST_BACKTRACE=full monolith ./test.html -o test2.html
file:///Users/huliax/projects/a_project/./test.html
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: ()', src/main.rs:164:9
stack backtrace:
   0:        0x10c6e49cc - <std::sys_common::backtrace::_print::DisplayBacktrace as core::fmt::Display>::fmt::h1216a26f3ce3c114
   1:        0x10c6fef9e - core::fmt::write::ha0ec7ad9e1aa7104
   2:        0x10c6e42b9 - std::io::Write::write_fmt::he79f4e513ba146b5
   3:        0x10c6dcc9a - std::panicking::default_hook::{{closure}}::hdf6a192f990c9642
   4:        0x10c6dc9a5 - std::panicking::default_hook::hc1d0cb86aebc9fbd
   5:        0x10c6dd378 - std::panicking::rust_panic_with_hook::hbe9b3056eff949a6
   6:        0x10c6dcf12 - rust_begin_unwind
   7:        0x10c6fe6af - core::panicking::panic_fmt::hf1009b33c3b398fd
   8:        0x10c6fe725 - core::result::unwrap_failed::h09cf7a39453b742a
   9:        0x10c4a7095 - monolith::main::hc0dfcfc2c23834aa
  10:        0x10c4a7456 - std::rt::lang_start::{{closure}}::h40ea679cda206d6a
  11:        0x10c6dcdd8 - std::panicking::try::do_call::h69374c200cb9b71d
  12:        0x10c6e8d9b - __rust_maybe_catch_panic
  13:        0x10c6d9113 - std::rt::lang_start_internal::h892855ab5abc97d0
  14:        0x10c4a7449 - main
```

`test.html` is as simple as possible:
```
<!DOCTYPE html>
<html>
  <head>
    <title>NotEmpty</title>
  </head>
  <body>
  Test
  </body>
</html>
```

**Comment by snshn** [comment]
Monolith does support both local fs paths and file: URLs, the error message seems to indicate that it wasn't able to write to test2.html. Could you please try running `RUST_BACKTRACE=full monolith ./test.html >> test2.html` instead, for comparison? And thank you for reporting this.

**Comment by mfaughn** [comment]
Same result.  I diff'ed the stack trace w/ the first one and only the addresses are different.

**Comment by snshn** [comment]
Thank you for checking that. From the log it seems that you're running monolith on macOS. May I ask if you installed monolith via brew or via cargo?

**Comment by mfaughn** [comment]
via brew.

Thanks for looking into this.

**Comment by amozlyw** [comment]
Just adding on, since I also faced this exact error while trying to run this on Windows using the compiled Windows binary from:https://github.com/Y2Z/monolith/releases v2.2.6.

```
C:\temp>monolith.exe "C:\temp\test.html"
file:///C:/temp/test.html
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: ()', src\main.rs:164:9
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

I tried downloading the monolith.exe for v2.2.2 and was able to run it successfully without any errors.

Hope that helps?

**Comment by snshn** [comment]
@mfaughn @amozlyw 
The bug has been fixed in [v2.2.7](https://github.com/Y2Z/monolith/releases/tag/v2.2.7), please give it a couple of days to become available via Homebrew. Thank you for notifying of the issue and helping me patch it.
